setup.py

```diff
@@ -47,7 +47,7 @@ def check_env_flag(name: str, default: bool = False) -> bool:
 
 # define install requirements
 install_requires_list = ['packaging', 'numpy', 'schema', 'pyyaml']
-opt_install_requires_list = ['neural_compressor', 'transformers']
```

Comment:
why peft is a mandatory here? should we make it optional when checking is_xx_available or install under fine-tuning usage?